src/main/java/edu/hm/hafner/analysis/descriptor/AcuCobolDescriptor.java

```diff
@@ -0,0 +1,59 @@
+package edu.hm.hafner.analysis.descriptor;
+
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.AcuCobolParser;
+
+/**
+ * A Descriptor for the AcuCobol warnings.
+ *
+ * @author Lorenz Munsch
+ */
+public class AcuCobolDescriptor implements Descriptor {
```

Comment:
It would make sense to move the `DescriptorFactory` to the `edu.hm.hafner.analysis.descriptor` package: then we could hide all descriptors as package private classes.